app/assets/javascripts/discourse/controllers/topic.js.es6

```diff
@@ -1035,6 +1036,20 @@ export default Ember.Controller.extend(bufferedProperty("model"), {
 
     resetBumpDate() {
       this.model.resetBumpDate();
+    },
+
+    removeTopicTimer(statusType, topicTimer) {
+      TopicTimer.updateStatus(
+        this.get("model.id"),
+        null,
+        null,
+        statusType,
+        null
+      )
+        .then(() => this.set(`model.${topicTimer}`, Ember.Object.create({})))
+        .catch(error => {
```

Comment:
one liner

Reply:
@jjaffeux I "de-one-lined" this bit, I'm guessing that's what you had in mind, LMK otherwise.

Reply:
I was talking of writing:

```
 .catch(error => popupAjaxError(error));
```

Reply:
Ah, right. Thanks!